fieldsolver/ldz_electric_field.cpp

```diff
@@ -1531,95 +1531,66 @@ void calculateElectricField(
    
    if (cellSysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE) return;
    
-   cuint cellSysBoundaryLayer = technicalGrid.get(i,j,k)->sysBoundaryLayer;
-   
-   if ((cellSysBoundaryFlag != sysboundarytype::NOT_SYSBOUNDARY) && (cellSysBoundaryLayer > 1)) {
-      // Sysboundary level 2+ cells
+   cuint mask = technicalGrid.get(i,j,k)->SOLVE;
```

Comment:
Perhaps this isn't technically a mask, it's a bitfield :) We're using e.g. compute::EX as a mask on it

Reply:
Better now?